src/arkode/arkode.c

```diff
@@ -272,6 +272,14 @@ int ARKodeSStolerances(void* arkode_mem, sunrealtype reltol, sunrealtype abstol)
     return (ARK_ILL_INPUT);
   }
 
+  /* Ensure that vector supports N_VAddConst */
+  if (!ark_mem->tempv1->ops->nvaddconst)
```

Comment:
I assume it doesn't matter but just want to double check this doesn't need to be `ark_mem->yn`.

Reply:
Either is fine -- this just needs to be a vector that was allocated inside `arkAllocVectors` (both `yn` and `tempv1` are allocated there)